### PR TITLE
Remove OpenShift without Service Discovery guide

### PR DESCRIPTION
**Pull request by nyechiel**
Signed-off-by: nyechiel <nyechiel@redhat.com>

### CODE REVIEW

File: src/content/quickstart/openshift/_index.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ weight: 20
 
 This quickstart guide covers the necessary steps to deploy two OpenShift Container Platform (OCP) clusters on AWS with
 full stack automation, also known as installer-provisioned infrastructure (IPI). Once the OpenShift clusters are deployed, we deploy
-Submariner to interconnect the two clusters. Note that this guide focuses on Submariner for network connectivity only. For Submariner with
-Service Discovery, please refer to the [Submariner with Service Discovery guide](./service_discovery/).
+Submariner with Service Discovery to interconnect the two clusters. Note that this guide focuses on Submariner deployment on clusters with
```
Comment (sridhargaddam) [inline src/content/quickstart/openshift/_index.md:9]: There is some issue while rendering the index. Can you please take a look.
Screenshot from 2020-11-05 13-53-22

**Review by sridhargaddam — CHANGES_REQUESTED** [review]

File: src/content/quickstart/openshift/_index.md (hunk 1 above)
Comment (nyechiel) [reply src/content/quickstart/openshift/_index.md:9]: @sridhargaddam yeah I've noticed that as well. If you roll over the "what seems to be just an empty space area" there, you will see that it actually has links to the shared pages, and you can even access them if you click on the URL. I am already looking at fixing this, but will do so in a different PR (note that this issue is seen with the current version of the website and is not something that this change has introduced).

Comment (sridhargaddam) [reply src/content/quickstart/openshift/_index.md:9]: I see, thanks @nyechiel

**Review by sridhargaddam — APPROVED** [review]

File: src/content/quickstart/openshift/_index.md (hunk 1 above)
Comment (dfarrell07) [reply src/content/quickstart/openshift/_index.md:9]: Yeah I had been wondering about this too, good job tracking down those details @nyechiel.

File: src/content/quickstart/openshift/_index.md [hunk 2]
```diff
@@ -35,17 +36,21 @@ customize the AWS instance type as shown below.
 
 {{< include "quickstart/openshift/run_prep_for_subm.md" >}}
 
-### Install `subctl`
+### Install subctl
```
Comment (dfarrell07) [inline src/content/quickstart/openshift/_index.md:39]: You don't think we should use `code style` in headings?

**Review by dfarrell07 — APPROVED** [review]

File: src/content/quickstart/openshift/_index.md (hunk 2 above)
Comment (nyechiel) [reply src/content/quickstart/openshift/_index.md]: Fixed

**Review by mangelajo — APPROVED** [review]

**Pull request merged**